chain-impl-mockchain/src/testing/builders/vote.rs

```diff
@@ -37,9 +43,10 @@ pub fn decrypt_tally(
                 .map(|member| member.secret_key())
                 .map(|secret_key| encrypted_tally.partial_decrypt(&mut thread_rng(), secret_key))
                 .collect::<Vec<_>>();
-            let tally = encrypted_tally
-                .decrypt_tally(max_votes, &decrypt_shares, &table)
-                .unwrap();
+            let validated_tally = encrypted_tally
+                .validate_partial_decryptions(&members_pks, &decrypt_shares)
+                .expect("Invalid shares");
```

Comment:
I think this should leverage some kind of error instead of just unwraping here. It may lead an attack in where  some fake shares are sent and then node just crash(?)

Reply:
This is testing code where we directly control committee members.